src/payment/klarna-setup.md

```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
+1. If your Klarna account supports B2B payments, set **Enable B2B** to `Yes`.
 
-     ![Klarna Payments]({% link images/images/config-sales-payment-methods-klarna-payments.png %}){: .zoom}
-     [_Klarna Payments_]({% link configuration/sales/klarna.md %})
+1. If you want Klarna to prepare a set of personalized credit options for each customer, set **Enable Data sharing** to `Yes`. Then, set **Enable Data sharing on load** to one of the following:
 
-1. If you want to stop here, tap <span class="btn">Save Config</span>.
+   |**No** |Customer billing and shipping address information is shared with Klarna after the transaction is authorized. If a shipping address error is encountered during authorization, the customer is instructed to return to the previous step and correct the problem.|
+   |**Yes** |Customer billing and shipping address information is shared with Klarna when the payment method is selected during the checkout process.|
 
-1. When prompted, click the [Cache Management]({% link system/cache-management.md %}) link at the top of the page.  Then, refresh all invalid caches.
+1. Enter a **Sort Order** number to determine the position of Klarna in the list of payment methods during checkout. (0 = first, 1 = second, 2 = third, and so on.)
 
 ## Step 3: Complete the Design Settings **(Optional)**
 
-You can complete the Design settings now, or at a later time. You might use a color picker to experiment with different colors for each element to match your store’s theme. The results might vary, depending on theme.
+1. Enter the hexadecimal value of the color that you want to use for the following elements. You can use the color chooser to experiment with different colors for each design element to match your store’s theme. The results might vary, depending on theme.
 
-Enter the hexadecimal value of the color that you want to use for the following elements.
+   - Details color
+   - Border color
+   - Selected border color
+   - Text color
+   - Border radius
 
-- Details color
-- Border color
-- Selected border color
-- Text color
-- Border radius
+   ![Klarna Payments Design]({% link images/images/config-sales-payment-methods-klarna-payments-design.png %}){: .zoom}
+   [_Design_]({% link configuration/sales/klarna.md %})
 
-![Klarna Payments Design]({% link images/images/config-sales-payment-methods-klarna-payments-design.png %}){: .zoom}
- [_Design_]({% link configuration/sales/klarna.md %})
+1. When complete, click <span class="btn">Save Config</span>.
 
-When complete, click <span class="btn">Save Config</span>. Then when prompted, click the [**Cache Management**]({% link system/cache-management.md %}) link at the top of the page to refresh all invalid caches.
+1. Click the [**Cache Management**]({% link system/cache-management.md %}) link in the message at the top of the page, and refresh all invalid caches.
```

Comment:
should this be "Click Cache Management in the..."?

Reply:
Yes, that's better.